pom.xml

```diff
@@ -75,7 +75,11 @@
             <groupId>org.springframework.boot</groupId>
```

Comment:
Good job! 
Small tip: The commit name "added pagination and swagger annotations" would be more descriptive.